rdr_service/resource/calculators/participant_ubr.py

```diff
@@ -233,3 +233,19 @@ def ubr_age_at_consent(consent_time, answer: (str, None)):
         if not 18 <= rd.years < 65:
             return UBRValueEnum.UBR
         return UBRValueEnum.RBR
+
+    @staticmethod
+    def ubr_overall(data: dict):
+        """
+        Calculate the UBR overall value from a dictionary of UBR values.
+        :param data:
+        :return: 0 or 1
+        """
+        result = UBRValueEnum.RBR
+        if data is None:
+            return result
+        # Test each UBR value for a UBR status.
+        for k, v in data.items():
+            if k.startswith('ubr_') and v == UBRValueEnum.UBR:
+                result = 1
```

Comment:
:nit: Further up on line 244 `UBRValueEnum.RBR` is used, but here an integer value is used instead. Would it make sense to use `UBRValueEnum.UBR`?

Reply:
Yes, I thought I got all those.  I'll make another commit.